projects_portfolio/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
```

Comment:
Overall good, I would recommend to play with responsiveness, as I can see, your idea was to create for three screen sizes, for example when width is 590px header starts behaving strange. Good work! ❤️

Reply:
Also about header, I would recommend to change it to have in this order as it it on the page : Home, About me, Projects and then Contacts